ci_framework/playbooks/99-logs.yml

```diff
@@ -32,10 +32,6 @@
       ansible.builtin.import_role:
         name: artifacts
 
-    - name: Run os_must_gather
```

Comment:
This block has been moved to `artifacts/tasks/main.yml` as part of log collection.